StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodEnums.swift

```diff
@@ -151,6 +153,8 @@ import Foundation
             return "Amazon Pay"
         case .mobilePay:
             return "MobilePay"
+        case .konbini:
+            return STPLocalizedString("Konbini", "Payment Method type brand name")
```

Comment:
Why don't we have a localized string for the other LPMs we converted from .dynamic? But added here?

Reply:
The previously .dynamic ones didn't use localized strings. Konbini is written differently in Japanese, though.